### DISCUSSION TITLE
Add method to invalidate indivisual tiles served from tile server like Martin

### DISCUSSION THREAD

**Discussion by Fabioni** [maplibre/maplibre-gl-js#4236] [Ideas]
Following https://github.com/maplibre/martin/discussions/1359#discussioncomment-9667183:

Hey, I have a PostGIS database and a Martin server serving vector tiles to Maplibre. The PostGIS data is updates frequently and I would like to show the updates immediately in Maplibre.
The only way for that I found so far is invalidating the complet vector map layer in maplibre.
Can we add a function to Maplibre to only invalidate specific tiles (or areas or zoom levels).

**Comment by HarelM** [comment]
What about cache control? Do you use it? It's not immediate, but I think you can fine tune the behavior...?

**Reply by Fabioni** [reply]
as far as I understand it, maplibre would not re-request the tile until you reload the page. Is that correct?

**Reply by HarelM** [reply]
That's not correct. maplibre-gl-js will refresh expired cache control tiles even if you don't refresh the page.
